### ISSUE TITLE
Try to file stream with raw data 2p nwb

### ISSUE THREAD

**Issue by rcpeene** [AllenInstitute/openscope_databook#30]
Once @Ahad-Allen uploads a 2p raw data file, we should try to see if this is a good use case for file streaming, or if some other system would be better. Additionally, we'll see if nwbwidget works for viewing these data or if we have to work on a way to view the video data.

**Comment by rcpeene** [comment]
Waiting on Ahad to generate new files with valid raw data in them. Should be done soon.

**Comment by jeromelecoq** [comment]
This is a duplicate of #54
